lvsfunc/source.py

```diff
@@ -0,0 +1,206 @@
+
+import os
+from abc import ABC, abstractmethod
+from io import BufferedReader
+from itertools import accumulate
+from pathlib import Path
+from subprocess import run
+from typing import Any, Callable, List, Optional, Tuple, Union, cast
+
+import vapoursynth as vs
+from pyparsedvd import vts_ifo
+from vsutil import is_image
+
+from .misc import get_matrix
+
+core = vs.core
+
+# List of containers that are better off being indexed externally
```

Comment:
Not sure why a bunch of empty lines were added in between these

Reply:
I don't know I just copied the old code